src/Microsoft.Identity.Client/PublicClientApplication.cs

```diff
@@ -146,7 +147,7 @@ public bool IsSystemWebViewAvailable
 
         /// <summary>
         /// Non-interactive request to acquire a security token from the authority, via Username/Password Authentication.
-        /// Available only on .net desktop and .net core. See https://aka.ms/msal-net-up for details.
```

Comment:
not sure why we want to remove this?

Reply:
removed it because it's available on the mobile platforms as well.